File: tools/dbench/answers/python_sonnet/s3/f1_users.py
valid = []
with open("users.txt") as f:
    for line in f:
        parts = line.split()
        if len(parts) == 3:
            try:
                valid.append(int(parts[2]))
            except ValueError:
                pass
print(len(valid))
print(sum(valid) // len(valid))

File: tools/dbench/answers/rust_haiku/s3/f1_users.rs
use std::fs;

fn main() {
    let txt = fs::read_to_string("users.txt").unwrap_or_default();
    let mut n = 0;
    let mut sum = 0;

    for line in txt.lines() {
        let parts: Vec<&str> = line.split_whitespace().collect();
        if parts.len() == 3 {
            if let Ok(age) = parts[2].parse::<i32>() {
                n += 1;
                sum += age;
            }
        }
    }

    println!("{}", n);
    if n > 0 {
        println!("{}", sum / n);
    } else {
        println!("0");
    }
}
